tensorboard/plugins/hparams/tf_hparams_session_group_details/tf-hparams-session-group-details.html

```diff
@@ -202,6 +202,14 @@ <h3>No metrics are enabled</h3>
           this._sessionGroupNameHash = tf.hparams.utils.hashOfString(
               this.sessionGroup.name);
         }
+        // Reset each scalar-card by prodding 'tag'.
```

Comment:
You could also choose to make `_resetDomain` a public method, such that
you can just call `c.resetDomain`. But this is fine if you prefer.

Reply:
Yes, that does seem more natural, right?
However, I don't think that would work here, since we need to reset the scalar-card only after it finished loading the data. Prodding the tag causes one of the implementing classes  (I don't remember which right now) to set a special flag to reset the domain once the load is completed.

Reply:
Ah, got it. Thanks for the explanation.